apps/iframe/src/Components/Modal/Modal.tsx

```diff
@@ -0,0 +1,84 @@
+import React, { useState, useEffect } from 'react'
+import './Modal.scss'
+
+import Header from '../Header/Header'
```

Comment:
Seems these components could be included directly in the modal as they are not used anywhere else, no?

Reply:
this was for future proofing, in the most recent push the Header is used for more than just basic rendering.

Reply:
Better make this first implementation straight-forward rather than trying to future-proof anything, these will likely change once integrated in self.id anyways.